src/app/pages/wallets/wallet-details/pages-wallet-details.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container walletDetailsContainer">
+
+  <div class="wallet__description">
+    <div class="name">
+      <div class="name__name">
+        {{ walletsDetailsData.data?.name }}
+      </div>
+      <div class="name__edit">
+        <button mat-stroked-button color="primary" (click)="handleWalletEdit(walletsDetailsData.data!)">Edit</button>
+      </div>
+    </div>
+
+    <div class="newTransaction">
+      <button mat-flat-button color="primary">Add new transaction</button>
+    </div>
+  </div>
+
+  <div class="wallet__sort">
+    <mat-form-field appearance="fill">
+      <mat-label>All Transactions</mat-label>
+      <mat-select [(value)]="selectedTransactionType" (selectionChange)="onTransactionTypeChange($event)">
+        <mat-option *ngFor="let transaction of walletTransactionType | keyvalue" [value]="transaction.value">
+          {{ transaction.value }}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+  </div>
+
+  <table *ngIf="walletsDetailsData.data" mat-table [dataSource]="(walletsDetailsData.data!.transactions | transactionsTypeFilter : selectedTransactionType) || []" class="mat-elevation-z8">
+
+    <ng-container matColumnDef="id">
+      <th mat-header-cell *matHeaderCellDef> No. </th>
+      <td mat-cell *matCellDef="let element"> {{element.id}} </td>
+    </ng-container>
+
+    <ng-container matColumnDef="date">
+      <th mat-header-cell *matHeaderCellDef> date </th>
+      <td mat-cell *matCellDef="let element"> {{element.date}} </td>
+    </ng-container>
+
+    <ng-container matColumnDef="description">
+      <th mat-header-cell *matHeaderCellDef> description </th>
+      <td mat-cell *matCellDef="let element"> {{element.description}} </td>
+    </ng-container>
+
+    <ng-container matColumnDef="amount">
+      <th mat-header-cell *matHeaderCellDef> amount </th>
+      <td mat-cell *matCellDef="let element">
+        <span [ngStyle]="{'color': element.amount < 0 ? 'red' : 'green'}">
```

Comment:
You should go with css classes like `amount--expense` and `amount--income` and store colors proper SCSS file with other color variables

Reply:
This was not changed so far.